### DISCUSSION TITLE
Chart ideas

### DISCUSSION THREAD

**Discussion by hburn7** [osu-tournament-rating/otr-web#194] [Ideas]
We have a large space where we want to put a chart but aren't quite sure. What would be a useful metric to track?

Criteria:
- If a bar chart, cannot have more than 30 items on the x-axis.
- Should not be overly dense
- Should display a single, useful piece of information that can change with time

Example idea:
- Bar chart of frequency of play at a certain hour

We could also display two small charts, similar in size to what is already there. Feel free to throw out any ideas.

![image](https://github.com/osu-tournament-rating/otr-web/assets/38370573/db8c4b34-879e-4a59-9b3a-53daec02e09a)

**Comment by SourMongoose** [comment]
Not exactly a new chart idea but I feel like the spider charts are not a great way of visualizing the mod statistics - I think it would be better if it was consolidated with the "Most played mods" section into a table containing play rate, win rate, and avg score

**Comment by m1ntleaf** [comment]
What about something like a bar chart of matches per month over the last year? I feel like this would be interesting to see when someone was most active and such. Rating over time could also be graphed, though it wouldn't be very interesting, probably. (updates a few times per month at most)

**Reply by hburn7** [reply]
Rating over time is already implemented, and the most important chart by far. Can you describe the chart idea a little more? I'm not sure what this bar chart would look like. Do you mean frequency of play? If so I feel like this wouldn't be very useful since most matches are concentrated to 2-3 days per week.

**Reply by m1ntleaf** [reply]
Yes, I meant play frequency. This could be monthly, though, to be more useful. For example, knowing you played the most matches over the summer can have some relevance
